benchmark-overhead/src/test/java/io/opentelemetry/util/NamingConventions.java

```diff
@@ -5,18 +5,22 @@
 
 package io.opentelemetry.util;
 
-/** An container to hold both the local and container naming conventions. */
+/** A container to hold both the local and container naming conventions. */
 public class NamingConventions {
 
-  public final NamingConvention container = new NamingConvention("/results");
-  public final NamingConvention local = new NamingConvention(".");
+  public final NamingConvention container = new ContainerNamingConvention("/results");
```

Comment:
Maybe you can modify the modifier from `public` to `private` here. There is same problem in the next line.

Reply:
> The `container` and `local` are referenced elsewhere.